From 08b617f9ea93bd6e3b07fb36f7be61ddffd6286c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?R=C3=A9mi=20Bernon?= <rbernon@codeweavers.com>
Date: Wed, 18 Jun 2025 08:56:16 +0200
Subject: [PATCH] comctl32: Fix class name case in manifest.

This shouldn't matter in theory but some tests check the case exactly
and there's no good reason to use a different casing.
---
 dlls/comctl32/comctl32.manifest | 4 ++--
 dlls/comctl32/tests/listbox.c   | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/dlls/comctl32/comctl32.manifest b/dlls/comctl32/comctl32.manifest
index 4c86137667e..8d37eeffc4d 100644
--- a/dlls/comctl32/comctl32.manifest
+++ b/dlls/comctl32/comctl32.manifest
@@ -6,9 +6,9 @@
     <windowClass>ButtonListBox</windowClass>
     <windowClass>ComboBoxEx32</windowClass>
     <windowClass>ComboLBox</windowClass>
-    <windowClass>Combobox</windowClass>
+    <windowClass>ComboBox</windowClass>
     <windowClass>Edit</windowClass>
-    <windowClass>Listbox</windowClass>
+    <windowClass>ListBox</windowClass>
     <windowClass>NativeFontCtl</windowClass>
     <windowClass>ReBarWindow32</windowClass>
     <windowClass>ScrollBar</windowClass>
diff --git a/dlls/comctl32/tests/listbox.c b/dlls/comctl32/tests/listbox.c
index 4012811473c..8a9db99f386 100644
--- a/dlls/comctl32/tests/listbox.c
+++ b/dlls/comctl32/tests/listbox.c
@@ -56,7 +56,7 @@ static void CALLBACK msg_winevent_proc(HWINEVENTHOOK hevent,
 
     /* ignore events not from a listbox control */
     if (!GetClassNameA(hwnd, class_name, ARRAY_SIZE(class_name)) ||
-        strcmp(class_name, WC_LISTBOXA) != 0)
+        stricmp(class_name, WC_LISTBOXA) != 0)
         return;
 
     msg.message = event;
-- 
GitLab

